src/abstractdataframe/show.jl

```diff
@@ -579,3 +579,18 @@ function showcols(io::IO, df::AbstractDataFrame) # -> Void
 end
 
 showcols(df::AbstractDataFrame) = showcols(STDOUT, df) # -> Void
+
+using Juno
+import Juno: Inline, LazyTree, Table, Row, strong
+
+const SIZE = 25
+
+@render Inline frame::AbstractDataFrame begin
+  width = min(size(frame, 2), SIZE)
+  height = min(size(frame, 1), SIZE)
+  header = map(x->strong(string(x)), names(frame)[1:width]')
+  body = hcat([frame[1:height,i] for i = 1:width]...)
```

Comment:
Splatting vectors of arbitrary length is generally not a good idea as it will trigger compilation of specialized methods without a clear gain. I think you can just use `Array(frame[1:height, 1:width])` here.